utils_linux.go

```diff
@@ -215,7 +216,7 @@ func (r *runner) run(config *specs.Process) (int, error) {
 		r.destroy()
 		return -1, err
 	}
-	tty, err := setupIO(process, rootuid, r.console, config.Terminal, r.detach)
+	tty, err := setupIO(process, rootuid, r.console, config.Terminal, r.detach || r.create)
```

Comment:
I don't think the create/start split should impact I/O detachment.  The new `create` case is just like the old handling (`run` in this PR), so you should be able to do `create --detach` or not as you see fit.  Folks calling this from the command line, etc., can always background their `create` process, do their setup, call `start`, and then foreground their `create` process.